server/startup/migrations/v253.ts

```diff
@@ -0,0 +1,9 @@
+import { Permissions } from '../../../app/models/server/raw';
```

Comment:
```suggestion
import { upsertPermissions } from '../../../app/authorization/server/functions/upsertPermissions';
```
And this is the import you should use to make the other suggested change work.

Reply:
Done! Thanks for the review!